hydromt/models/model_api.py

```diff
@@ -883,8 +914,10 @@ def setup_maps_from_raster_reclass(
         rename: Optional[Dict] = dict(),
         **kwargs,
     ) -> List[str]:
-        """
-        HYDROMT CORE METHOD: Add data variable(s) to maps object by reclassifying the data in ``raster_fn`` based on ``reclass_table_fn``.
+        """Add data variable(s) to maps object.
```

Comment:
Can you undo the changes here? HYDROMT CORE METHOD should be here and for the API docs online as the first line is printed the rest is needed to easily differentiate between setup_maps_from_rasterdataset and setup_maps_from_raster_reclass